src/new/wallet.js

```diff
@@ -342,11 +354,23 @@ class HathorWallet extends EventEmitter {
    *
    * @param {Object} data Full transaction data
    *
-   * @return {Promise} Promise that resolves when transaction is sent
+   * @return {Object} bject with {success: true, sendTransaction, promise}, where sendTransaction is a
+   * SendTransaction object that emit events while the tx is being sent and promise resolves when the sending is done
    **/
   sendPreparedTransaction(data) {
```

Comment:
We should add an option to get the `SendTransaction` object instead of the promise.

Reply:
This method returns an object (in case of success) with `{'success': true, promise, sendTransactionObject}`, so who is using it can decide if want to get the feedback events, or the promise.

Reply:
Great! Thanks!